_data/changelogs/component-date-picker.yml

```diff
@@ -2,6 +2,15 @@ title: Date picker
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added readonly attribute to aria-disabled.
+    summaryAdditional: Now, the input element and calendar icon remain discoverable but disabled behaviors are replicated.
+    affectsAccessibility: true
+    affectsJavascript: true
+    affectsMarkup: true
```

Comment:
Since the markup changes come from the JS, and are not something the user has to manually add to their markup, we can remove this line.

```suggestion

```

Reply:
Resolved in [fefb694](https://github.com/uswds/uswds-site/pull/2794/commits/fefb6949907954544486e519967cf3c9c5b7f530)